static/app/utils/replays/createUrlToShare.tsx

```diff
@@ -0,0 +1,9 @@
+function createUrlToShare(currentTimeMs: number) {
+  const url = new URL(window.location.href);
+  if (Number.isInteger(currentTimeMs) && currentTimeMs > 0) {
```

Comment:
@SDaian `currentTimeMs` we can assume is a number (because of typescript) - but when we divide by 1000 it is no longer an integer. We probably only want whole numbers for `t` param.

Reply:
hey @billyvg you were right, added the following before calculate:

`const curentTimeSec = Math.floor(msToSec(currentTimeMs) || 0);`

what do you think?